BlackMarket/Networking/Services/ProductServices.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ProductsService.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 28/09/2023.
+//
+
+import Foundation
+
+internal class ProductServices: ProductServicesProtocol {
+  
+  private let apiClient: RSApiClient
+  
+  enum ProductError: Error {
+    case noProductsFound
+    case noPictureFound
+  }
+  
+  init(
+    apiClient: RSApiClient = RSApiClient.shared
+  ) {
```

Comment:
As a rule of thumb, please don't split lines if not needed for the line-lenght rule.
```suggestion
  init(apiClient: RSApiClient = RSApiClient.shared) {
```